modules/videoio/src/cap_interface.hpp

```diff
@@ -221,6 +221,30 @@ class IVideoCapture
     virtual int getCaptureDomain() { return CAP_ANY; } // Return the type of the capture object: CAP_DSHOW, etc...
 };
 
+static void applyMetadataRotation(const IVideoCapture& cap, OutputArray mat)
```

Comment:
I would prefer to split definition (here) and declaration (e.g., in cap.cpp file)

Reply:
Will do.